src/DiffSharp.Core/DiffSharp.Shorten.fs

```diff
@@ -0,0 +1,60 @@
+// Pipelined operations
+module DiffSharp.Shorten
```

Comment:
Maybe "DiffSharp.Abbreviations" though either is ok

Reply:
I've been thinking about many names. At some point it was "Shorthand". Then I shortened the shorthand to this.

Reply:
I thought about "Abbreviations" too, but it felt too long. "Abbrev" would not be great either.